packages/public/@babylonjs/viewer/package.json

```diff
@@ -35,6 +35,8 @@
         "@babylonjs/loaders": "^7.23.1"
     },
     "devDependencies": {
+        "@babylonjs/core": "^7.53.3",
+        "@babylonjs/loaders": "^7.53.3",
```

Comment:
Are these versions bumped automatically? Also should the ones in peerDependencies be bumped? I remember talking about that when we first setup the Viewer build, but I'm not sure if we want these dependencies to be in sync to be consistent with other babylon libs?

Reply:
and no, peer dependencies should only be bumped when you think it is needed. i.e. - when you use new APIs that are not available in a previous version. peer dependencies are always updated with major version. so when 8.0.0 will be released, the peer dependency version will be `^8.0.0`